src/app/personside/infotabs/dyplenkeTest/meldingerDyplenker.test.tsx

```diff
@@ -41,7 +41,7 @@ test('bytter til riktig tab og setter fokus på riktig melding ved bruk av dyple
 function clickOnMeldingerIOversikt(infoTabs: ReactWrapper) {
     infoTabs
         .find('.' + meldingerTest.oversikt)
-        .at(2)
         .find('button')
+        .at(1)
```

Comment:
Blir vel ikke like generisk som før, men det er kanskje helt ok?

Reply:
Jo, tror det, ble litt forvirra egentlig, for at(1) gir nå element nummer 2, mens den før ga element nr 1. 😕  Men den tester faktisk på samme entry som før